### PR TITLE
tutorial docs cleanup - step_05

### PR DESCRIPTION
**Pull request by modcoms**
adding step_06 as well

### CODE REVIEW

File: .docs/angular-meteor/client/views/steps/tutorial.step_05.html [hunk 1]
```diff
@@ -97,7 +97,7 @@
 Meteor scans all the HTML files in your directory for three top-level elements: HEAD, BODY, and TEMPLATE.
 The head and body sections are separately concatenated into a single head and body, which are transmitted to the client on initial page load.
 
-Meteor converts both the templates and body content and generates a client-side JavaScript file (bundle-time, with htmlJS) and makes the template available under the Template namespace.
+Meteor converts both the templates and body content and generates a client-side JavaScript file (bundle-time, with html & JS) and makes the template available under the Template namespace.
```
Comment (Urigo) [inline .docs/angular-meteor/client/views/steps/tutorial.step_05.html]: This is the HTMLJS library.  maybe instead it's better to add a link to the package inside Meteor?

Comment (Urigo) [inline .docs/angular-meteor/client/views/steps/tutorial.step_05.html]: @modcoms

**Comment by modcoms** [comment]
Yes, that makes it much clearer as to how Meteor is doing the bundling. I've amended the pull request with a link to the atmosphere package. Let me know what you think.

Martin

**Comment by Urigo** [comment]
thanks Martin!

**Pull request merged**